tiers/master.json

```diff
@@ -1415,39 +1415,79 @@
 			"tip": "May the bird gods bless you with evil chicken pieces",
 			"wikiLink": "https://oldschool.runescape.wiki/w/Collection_log#Miscellaneous",
 			"imageLink": "https://oldschool.runescape.wiki/images/thumb/Dragon_full_helm_detail.png/277px-Dragon_full_helm_detail.png",
-			"displayItemId": 11335
+			"displayItemId": 11335,
+			"verification": {
+				"method": "collection-log",
+				"itemIds": [
+					21509, 19707, 11335, 22103, 22100, 20439, 20436, 20442, 20433, 24034, 24037,
```

Comment:
Herbi (21509) is a pet, I'm not sure if this is where it belongs. Need to verify with the mods.